x-pack/plugins/enterprise_search/public/applications/app_search/components/analytics/analytics_layout.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+import { useValues, useActions } from 'kea';
+
+import { KibanaLogic } from '../../../shared/kibana';
+import { FlashMessages } from '../../../shared/flash_messages';
+import { Loading } from '../../../shared/loading';
+
+import { LogRetentionCallout, LogRetentionOptions } from '../log_retention';
+
+import { AnalyticsLogic } from './';
+import { AnalyticsHeader, AnalyticsUnavailable } from './components';
+
+interface Props {
+  title: string;
+}
+export const AnalyticsLayout: React.FC<Props> = ({ title, children }) => {
```

Comment:
This component is a refactor/change from ent-search. Currently the standalone UI is using 2 main layout/page views:

- Analytics.tsx
- AnalyticsSectionContent.tsx

There is a significant amount of repeated code between the two page components that I sought to DRY out with a single AnalyticsLayout wrapper. I hope you'll agree that it's an improvement 🙏

I'm definitely hoping at the very minimum the split between having a `analytics/components/` and `analytics/views/` subfolder will make the `analytics/` folder easier to navigate - right now in ent-search it's fairly hard for me to parse heh

Reply:
I do think this is an improvement, nice work.